package.json

```diff
@@ -38,10 +39,15 @@
   },
   "devDependencies": {
     "babel-core": "5.8.21",
+    "babel-eslint": "^4.0.5",
     "babel-loader": "5.3.2",
     "babel-relay-plugin": "^0.1.2",
     "del": "^1.2.0",
     "envify": "^3.4.0",
+    "eslint": "^1.1.0",
+    "eslint-plugin-react": "^3.2.2",
+    "eslint-plugin-react-internal": "file:eslint-rules",
```

Comment:
This actually requires npm v2+ so we'll need a `npm install -g npm@2` in the `.travis.yml` `before_install` hook.